### PR TITLE
New serverless pattern - dynamodbstream to lambda with stepfunction enrichment

### PR DESCRIPTION
**Pull request by moitrasayan007**
*Issue #, if available:*

*Description of changes: Dynamodbstream to lambda with stepfunction enrichment* 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by moitrasayan007** [comment]
It looks like the first check is failing due to 

`Policies:
        - PolicyName: CloudWatchLogs
          PolicyDocument:
            Version: '2012-10-17'
            Statement:
              - Effect: Allow
                Action:
                  - 'cloudwatch:*'
                  - 'logs:*'
                Resource: '*'`

The step function role uses this for logging and monitoring.

File: eventbridge-pipes-dynamodbstream-to-lambda-with-stepfunction-enrichment/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+Once the deployment has completed, you can test by
+
+
+```bash
+aws dynamodb put-item \
+    --table-name SampleTable \
+    --item PK={S="Message#1234"},SK={S="Channel#1231"},messageId={S="3"}
+```
+
```
Comment (julianwood) [inline eventbridge-pipes-dynamodbstream-to-lambda-with-stepfunction-enrichment/README.md:62]: Can you  please add a step where to check that the whole flow has completed succesfully.

File: eventbridge-pipes-dynamodbstream-to-lambda-with-stepfunction-enrichment/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+## Cleanup
+ 
+1. Delete the stack
+    ```bash
+    sam delete
+    ```
+
+----
+Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (julianwood) [inline eventbridge-pipes-dynamodbstream-to-lambda-with-stepfunction-enrichment/README.md:71]: 2023  please

Comment (moitrasayan007) [reply eventbridge-pipes-dynamodbstream-to-lambda-with-stepfunction-enrichment/README.md]: done Julian.

**Pull request merged**